src/main/webapp/sass/_core.scss

```diff
@@ -2989,11 +2989,13 @@ div.border {
 	}
 }
 #body {
-	.dropdown-menu>li>a {
-		color: $synapse-dark-text;
+	.dropdown-menu>li {
+		>a:not(.btn), >span {
```

Comment:
increase specificity so that this style is used, and also target any spans (these are used for menu items that are not clickable).